tests/unit/channel_test.py

```diff
@@ -237,7 +237,7 @@ def test_buffer_frame(self):
     def test_process_frames_when_no_frames(self):
         # Not that this should ever happen, but to be sure
         c = Channel(None, None, {})
-        stub(c.dispatch)
+        expect(c.dispatch).times(0)
```

Comment:
You don't need this, that's essentially what `stub` is doing. Do you prefer this usage?